### ISSUE TITLE
Silicon quest line overhaul for the upcoming changes

### ISSUE THREAD

**Issue by Nilau1998** [GTNewHorizons/GT-New-Horizons-Modpack#10347]
### Your GTNH Discord Username

Noila#7604

### Your Pack Version

2.1.2.4

### Your Proposal

The silicon line hasn't really been implemented yet into the quest book and therefore still has wrong ordering, e.g. raw silicon ingots being right at the beginning and bouls coming after that. I've put some thought into the whole thing with chochom#0271 and we came to the conclussion that the quests could look like the following:
![IQqZiF0](https://user-images.githubusercontent.com/64710705/168487387-8107bb0b-3b1d-4fee-81f4-039c0772303b.png)
The kanthal part on the right of the quest book got shifted to the left and made dependent on the MV hatches as these are required for kanthal, upon that follows the ability of making raw silicon ingots (because it needs kanthal coils now to make the ingots) and the rest of the story finally giving the ability to make HV circuits.

### Your Goal

The quests are still in work and this is a proposal for how they could look, or rather in which order they could appear.

### Your Vision

It implements the silicon line, or rather what has been done so far for it, into the quest book.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by chochem** [comment]
I would add that:
- we need new quests for the silicon boule (with some details, this is still a relatively early tier)
- we need to update the quests for glowstone doped and naq doped boules in later tiers.

regarding the picture the only two small adjustments I would make is
- make the coolant and kanthal cooling and advanced chem bath quests main quest.
- move the advanced chem bath quest to the left as well to remove those lines going across everything (just next to mixer maybe?). Sorry I thought you had already done that when you showed it to me earlier.

but that can be done once you/somebody actually puts this all in.

**Comment by Nilau1998** [comment]
I had the chemical bath next to it but selected the wrong image for the upload.. here is an updated image with the proposals you mentioned:
![](https://i.imgur.com/W2TQJRt.png)

**Comment by henrydukepickle** [comment]
Reminding y'all that SMD transistors are available in MV and can be used to bypass the need for Si ingots in MV entirely.

**Comment by chochem** [comment]
yeah thanks for noticing! That complicates things for the quests obviously. Basically SMD transistors (needs Gallium!) should be an alternative route players can choose.

**Comment by Shadowsoftime99** [comment]
Making a quick note in here, that the later tiered boule quests (naquadah, etc) will need to be updated to match. Pretty much just swapping out the current asks of raw silicon blocks for the new refined ones

**Comment by chochem** [comment]
completed with https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/10753
